corehq/apps/app_manager/models.py

```diff
@@ -6247,6 +6247,25 @@ def to_json(self, app_names):
         return LatestEnabledBuildProfileSerializer(self, context={'app_names': app_names}).data
 
 
+class ApplicationReleaseLog(models.Model):
+    domain = models.CharField(max_length=255, null=False, default='')
+    user_email = models.CharField(max_length=255)
```

Comment:
Curious why this is being stored, instead of being looked up when the logs are pulled. Generally we shouldn't be storing denormalized data. (Domain is arguably redundant, but it's so useful for sql dump and domain deletion.)

Reply:
> Generally we shouldn't be storing denormalized data

@orangejenny Can you expand on this more? 

Moreover, I think the intent here was to avoid the lookup though we can use `cached_user_id_to_user_display` as well.

Reply:
I mean that we shouldn't store data that con be looked up based on other data, since it can get out of sync. (Admittedly, in this case I'm not 100% certain a user can change their email address, since we use it for username, so I mean this as a general best practice.)

I agree about using `cached_user_id_to_user_display`, relying on code solutions like caches is usually better than storing more data.

Reply:
@orangejenny @mkangia thanks for the input, i will make the changes.